sky/execution.py

```diff
@@ -348,7 +352,7 @@ def launch(
 
 @usage_lib.entrypoint
 def exec(  # pylint: disable=redefined-builtin
```

Comment:
Aside: btw, `sky.exec(my_dag, cluster_name=my_cluster)` doesn't really make sense, because presumably an n-node DAG will have n clusters.

Reply:
Good point! For backward campatibility, we may need to keep this for now. Added a warining for the deprecation. Wdyt?

Reply:
Yep, mentioned this for future redesign.